Homescreen fix post

ShadowMuffin25 opened the pull request:
Homescreen: Changed the userEffect() to handleSave() function and add a Save button (so that it wouldn't fetch/save everytime a single icon is changed). Keep track of changes in homescreen (setHasChange()) for the Save button (the button will be gray if nothing changes)
PeriodCalendarScreen: Changed the renderWeekly() to print out data easier, and change the moods line to fix the bug (not seeing moods in Week section)
gitignore files: Changed to ignore my virtual env

ShadowMuffin25 committed:
Add the Save button

ShadowMuffin25 committed:
Add the Save button

ShadowMuffin25 committed:
Add save button in home and fix moods in calendar

nhan0504 committed:
Change the flash screen image (#73)

# What
- The flash screen image say App Name during app load when first open up
- Change the image to show the app name "Sukoon"

ShadowMuffin25 committed:
Add save button in home and fix moods in calendar

ShadowMuffin25 committed:
Delete myenv_310 directory

ShadowMuffin25 committed:
Change the save button

nhan0504 commented on mobile/screens/HomeScreen.js:
```diff
@@ -507,6 +496,15 @@ const HomeScreen = () => {
+                <View className="mt-3 px-3 mb-6">
+                    <FormButton 
+                        disabled={!hasChanges || isLoadingButton} 
+                        btnTitle={isLoadingButton ? "Saving..." : "Save"}
```
Instead of hardcoding button text like "Save" use the translation like like this `i18n.t('home.save')` If there is not translation for the text you want you can add it

ShadowMuffin25 committed:
Remove venv

ShadowMuffin25 committed:
Delete myenv_310 directory

nhan0504 committed:
Change the save button

nhan0504 committed:
Merge branch 'homescreen-fix-post' of https://github.com/ducth1903/sas-period-tracker into homescreen-fix-post

nhan0504 committed:
Refactor button highlight styling and use solid color instead of gradient

nhan0504 commented on mobile/screens/PeriodCalendarScreen.js:
```diff
@@ -215,10 +215,11 @@ const PeriodCalendarScreen = ({ props }) => {
-                date: weekDates[weekDaysEnglish.indexOf(day)],
-                data: getDayData(weekDates[weekDaysEnglish.indexOf(day)])
-            }
+            weekDaysEnglish.forEach((day) => {
```
Instead of using the `weekDaysEnglish` array, can use the weekday translation. But it's working now so can do the refactor in a different PR

nhan0504 reviewed (changes requested):
Minor changes for the language translation but otherwise LGTM!

vitdoan committed:
SAS-65 Refactor Code and Fix Scroll Bar Overide (#68)

Fix these 2 issues: 
- Text gets cut off horizontally on some articles, specifically ones
with images embedded.
- Scroll icon override text.

nhan0504 committed:
Revert "SAS-65 Refactor Code and Fix Scroll Bar Overide (#68)" (#76)

This reverts commit d4735e0155770be53cce9688b7376c91c72e50e2.

Too complicated conflict with #75 . Need to revert #68

ShadowMuffin25 committed:
Add in translation for save button

nhan0504 reviewed (approved):
LGTM!
